website/versioned_docs/version-v3.15.x/install.md

```diff
@@ -0,0 +1,117 @@
+---
+id: install
+title: Installation
+---
+
+## Prerequisites
+
+### Minimum Kubernetes Version
+
+The minimum supported Kubernetes version for Gatekeeper is aligned with the Kubernetes releases listed in the [Kubernetes Supported Versions policy](https://kubernetes.io/releases/version-skew-policy/). For more information, please see [supported Kubernetes versions](https://github.com/open-policy-agent/gatekeeper/blob/master/docs/Release_Management.md#supported-kubernetes-versions).
+
+**Note:** Gatekeeper requires resources introduced in Kubernetes v1.16.
+
+### RBAC Permissions
+
+For either installation method, make sure you have cluster admin permissions:
+
+```sh
+  kubectl create clusterrolebinding cluster-admin-binding \
+    --clusterrole cluster-admin \
+    --user <YOUR USER NAME>
+```
+
+## Installation
+
+### Deploying a Release using Prebuilt Image
+
+If you want to deploy a released version of Gatekeeper in your cluster with a prebuilt image, then you can run the following command:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/v3.16.0-beta.0/deploy/gatekeeper.yaml
```

Comment:
Shouldn't this be v3.15.0? @sozercan

Reply:
may be `NEW_VERSION` in [this](https://github.com/open-policy-agent/gatekeeper/pull/3178/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R464) fix still points to beta? 🤔

Reply:
@JaydipGabani can you pls push an update for this PR to address these?

Reply:
Sure!